component-maps.yml

```diff
@@ -288,6 +288,12 @@ git:
     release_component: false
     independent_component: true
 
+  app-update-module:
+    docker_image: []
+    docker_container: []
+    release_component: true
```

Comment:
We will not release it together with the rest of Mender. So this one needs to be `false`.

As per today we are in early steps to redesign completely the release process, and `components-maps` as we know it will change substantially. But for now we need to deal with what we have :)